src/vscode-bicep/package.json

```diff
@@ -408,6 +408,7 @@
         },
```

Comment:
Rather than hacking this for one particular command, let's discuss a consistent experience across all commands...  Personally I've always thought we should *always* use the current editor if it's bicep, but that change should be for all commands.

BTW, did you try adding a max results to findFiles?

Reply:
Sure, if we can come to a consensus that works for other commands, that's ideal! I also think it's perfectly fine to have different behavior for different commands - e.g.:
* Commands which operate on a cursor position within a file: this one, any future refactoring commands we may add
* Commands which operate on a whole file: Deploy, Build etc.

My priority is to restore the original behavior of this command: active cursor + insert command => no file IO - not too fussed how we achieve that!

Reply:
Exactly my sentiments (vetoed by previous PM).